third_party/tonic/logging/dart_error.cc

```diff
@@ -10,28 +10,15 @@
 namespace tonic {
 namespace DartError {
 const char kInvalidArgument[] = "Invalid argument.";
-DartPersistentValue on_error;
-UnhandledExceptionReporter log_unhandled_exception = [](const std::string&,
-                                                        const std::string&) {};
+UnhandledExceptionReporter log_unhandled_exception = [](Dart_Handle,
```

Comment:
nit: this should be in an anonymous namespace to avoid any collisions, same with the thing above it.

Reply:
The thing above it is existing code that's used from at least one other TU and is public.

Reply:
I misread this.